product/quickstart/customizing-the-navbar.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: "Customizing the navbar"
+description: "Learn how to work with global components in Makeswift."
+icon: "rectangle-wide"
+---
+
+[Global Components](/product/components#global-components) are groups of components that can be saved and reused across your entire site. This is a perfect use case for sections that should look the same across all pages like navbars, footers, etc. In fact, the **Navbar** component in the **Venture** template is a Global Component.
+
+## Editing the navbar
+
+In the top part of the canvas, select the **Navbar** component. When you click on it, notice you don't immediately see its properties in the Properties Sidebar. Since this is a Global Component, you must enter edit mode to make changes.
+
+<Frame>
+  ![ Global component edit
+  button](/images/quickstart/global-component-edit-button.jpg)
+</Frame>
+
+With the **Navbar** component selected, click **Edit component** in the Properties Sidebar to enter edit mode. You'll now see the properties associated with this global component. In the canvas, click on the nested component with the name **Navigation**. Here you can edit the links in the Navbar component.
```

Comment:
The image below shows the wrapper Box selected. I was thinking it might be clearer if the image showed the Navbar component selected and the list of links that are currently there in the Properties Panel